engineyard/ey-cookbooks-stable-v5
# Eng-1145 Ey Services Config Inclusion

Description of your patch
--------------

Update resin to include Description of your patch



Recommended Release Notes
--------------
This update resolves an issue with the ey config gem file

Estimated risk
--------------

High


Components involved
--------------

Resin

Description of testing done
--------------

1. Boot stack under V5 with resin
1. Verify chef runs correctly
1. Verify deploy finishes
1. Add an add on to the environment
1. Deploy
1. Verify that the ey-services-config.yml is created in the shared directory

QA Instructions
--------------
1. Boot stack under V5 with resin
1. Verify chef runs correctly
1. Verify deploy finishes
1. Add an add on to the environment
1. Deploy
1. Verify that the ey-services-config.yml is created in the shared directory
